scripts/apps/monitoring/services/CardsService.ts

```diff
@@ -242,6 +242,9 @@ export function CardsService(search, session, desks) {
             criteria.repo = card.search.filter.query.repo;
         } else if (desks.isPublishType(card.type)) {
             criteria.repo = 'archive,published';
+            if (card.type === 'deskOutput') {
+                query.filter({not: {term: {state: 'unpublished'}}});
```

Comment:
What the purpose of this? To exclude unpublished items from being displayed in the output stage? If so, I'd say it'd be better to exclude it in the backend.

Reply:
Both search and deskOutput call the search api so if we filter the unpublish items from the backend then when we try to search also, it will not show the unpublished items in the search.